### PR TITLE
Improve wording of the introduction 

### PR DESCRIPTION
**Pull request by kslstn**
Hi there, this is what I've done:

- Make text more concise
- Move feature list up
- Remove tip as its content was partly in the body text already.
- Improve grammar 

### What is accomplished by your PR?
I hope now the Vue Starter is more attractive to people first learning about it via the docs.

### Is there something controversial in your PR?

- I used 'Vue Starter' as a project name, but I've seen vue-starter and Vue.js Starter elsewhere.
- Part of the text is in the first person. This makes it authentic, but the I in the story is never introduced. Let's consider adding your name at the bottom or make it about the reader: "You know in how every project you started with libraries at some point you just get stuck?". I'd say other than in the very first bit of the introduction, the first person singular should not be used. After all, the project is supposed to be maintained by the community. 
- Readers are addressed in the third person ("Vue Starter is for experienced developers [...]") and in the second person ("You'd like to focus on developing skills [...]"). This is a bit awkward to read. For a sense of community, I find the second person more attractive.

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -4,28 +4,13 @@ next: ./guide/install
...
-library and you can add any additional use case you encounter. 
-
-::: tip 
-The limitations of your project are up to the imagination of your designers and your ability to implement their feature requests
-:::
-
-::: warning
-I would not recommend this starter for beginners, **this starter is for experienced developers who need to deliver projects very quickly with the highest possible quality**.
-:::
+I often had to write feature-requests for libraries, remove the Webpack configuration at the start of the project, or had to create a local fork of the library to implement the required feature. To bring an end to that kind of mess, I decided to take all common use cases that I've encountered and bring them together in this starter. It's not a library. The **The code is all yours** to modify and build upon.
```
Comment (devCrossNet) [inline docs/introduction.md]: remove has to be replaced with "eject", this is the term that e.g. create-react-app or angular-cli uses. It means that the webpack config that is used behind the scenes of this tools, is made accessible to the developer to modify it.

Comment (devCrossNet) [reply docs/introduction.md]: > To bring an end to that kind of mess

can we come up with a nicer wording for this? It is not the kind of tone that I usually use.

**Comment by devCrossNet** [comment]
> I used 'Vue Starter' as a project name, but I've seen vue-starter and Vue.js Starter elsewhere.

I will check this today, I think we should stick to `vue-starter` consistently.

> Part of the text is in the first person

Yes, I think you're completely right, it should be consistent with the `second person` as you suggested

Would be really good if you could apply this changes to the introduction page, I can then adapt it to the other pages if this would be too much work for you 👍

**Comment by devCrossNet** [comment]
What do you think about this?
I adapted your changes and left parts that I really like from the old version, but rewrote it to 2nd person.

```
---
prev: ./
next: ./guide/install
---
# Introduction

## This is not just another starter project


When starting the development of a new project, it's often tempting to start with some libraries.
They provide solutions to already solved problems, so you can solve on the unique challenges of your application.

But you know that this only works to a certain degree. In most projects you are working on, the number of phrases like 
**_"I can not change that because the library does not support it."_** increases over time.

You often been in situations where you had to write feature-requests, eject the webpack configuration in the first second of the project,
or you have to create a local fork of the library to implement the requested features.

Because you're not the only one with this experience,
we decided to implement all the use cases, that we experienced over time, in this starter.

It's not a library.

**The code is all yours** to modify and build upon. 

::: tip No Excuses
The limitations of your project are up to the imagination of the designers in your project and your ability to implement their feature requests.
:::

::: tip Productivity Tip
The key for high-performance engineering is a good balance between a variety of music genres,
for this reason, we provide you with a [Beast mode playlist on Youtube](https://www.youtube.com/playlist?list=PLcLlpvN8RJznCOm4dJ5rN5VJKFfy0hMNp)
:::

::: warning
**Use this project if you are an experienced developer who needs to deliver projects quickly with the highest possible quality.**

If you look for an easier way to get started with Vue and Webpack, 
[this template](https://github.com/vuejs-templates/webpack) may be more your cup of tea.
:::

## Features

- Quick scaffolding with the [Generator CLI](./guide/cli.md)
- Enterprise ready
- SEO with [vue-meta](https://github.com/declandewet/vue-meta)
- Industry-standard i18n support with [vue-i18n](https://github.com/kazupon/vue-i18n)
- Progressive-Web-App
- Component library
- Routing with data prefetching
- [Storybook integration](https://storybook.js.org/basics/guide-vue/)
- Predictable state management with [vuex](https://vuex.vuejs.org/en/)

## Todo

This is an every evolving project,
if you want to help or you have an idea how to improve this project - 
**feel free to file an [issue](https://github.com/devCrossNet/vue-starter/issues)**

## Why not vue-cli or nuxt.js ?

This boilerplate is opinionated, aiming to educate and empower users to essentially configure their own framework,
ideally suited to their needs. It also includes as few libraries as possible to keep you flexible and to enable you
to implement every product requirement that comes down the road.

Use cases for this boilerplate:
- The requirements for the product might change and you want to maintain maximum flexibility and control.
- You'd like to focus on developing skills that will transfer across any Vue project.
- You're working in a large team, so you need tooling to help everyone avoid common mistakes,
  write in a consistent style, and avoid bikeshedding in PRs.
```

**Comment by kslstn** [comment]
Nice! I think this sentence was supposed to start with "You've often been":

> You often been in situations where you had to write feature-requests, eject the webpack configuration in the first second of the project,
> or you have to create a local fork of the library to implement the requested features.

I notice some superfluous line breaks. I guess because you text editor is set up like that, but in viewers like this website, the line breaks just appear mid-sentence.

**Comment by kslstn** [comment]
Ah I closed the PR. Well you know I believe it's my very first on Github, so I don't really know what that means anyway.

**Comment by devCrossNet** [comment]
Don't worry about it, I will take your suggestions and merge it into master later today. Thank you very much 😺

**Pull request closed without merging**